### PR TITLE
JDK-8328115 : Convert java/awt/font/TextLayout/TestJustification.html applet test to main

### PR DESCRIPTION
**Pull request by honkar-jdk**
java/awt/font/TextLayout/TestJustification.html  is converted to main manual test using PassFailJFrame. 

This test was previously problemlisted on macOS. When tested on macOS, the problem still exists and not all lines are justified to the same length (lines containing Arabic & Hebrew differ by a small amount). For this reason, the test is retained in problemlist and the file extension in problemlist is changed to .java to match the test changes.

---------
### Progress
- [x] Change must be properly reviewed (1 review required, with at least 1 [Reviewer](https://openjdk.org/bylaws#reviewer))
- [x] Change must not contain extraneous whitespace
- [x] Commit message must refer to an issue

### Issue
 * [JDK-8328115](https://bugs.openjdk.org/browse/JDK-8328115): Convert java/awt/font/TextLayout/TestJustification.html applet test to main (**Bug** - P4)

### Reviewers
 * [Alexey Ivanov](https://openjdk.org/census#aivanov) (@aivanov-jdk - **Reviewer**)

### Reviewing
<details><summary>Using <code>git</code></summary>

Checkout this PR locally: \
`$ git fetch https://git.openjdk.org/jdk.git pull/18285/head:pull/18285` \
`$ git checkout pull/18285`

Update a local copy of the PR: \
`$ git checkout pull/18285` \
`$ git pull https://git.openjdk.org/jdk.git pull/18285/head`

</details>
<details><summary>Using Skara CLI tools</summary>

Checkout this PR locally: \
`$ git pr checkout 18285`

View PR using the GUI difftool: \
`$ git pr show -t 18285`

</details>
<details><summary>Using diff file</summary>

Download this PR as a diff file: \
<a href="https://git.openjdk.org/jdk/pull/18285.diff">https://git.openjdk.org/jdk/pull/18285.diff</a>

</details>

### Webrev
[Link to Webrev Comment](https://git.openjdk.org/jdk/pull/18285#issuecomment-1995388780)

### CODE REVIEW

**Review by aivanov-jdk — APPROVED** [review]
Looks good except for a minor comment.

**Comment by aivanov-jdk** [comment]
> This test was previously problemlisted on macOS. When tested on macOS, the problem still exists…

I wonder if it depends on the font used. Is the selected font available on macOS?

**Comment by honkar-jdk** [comment]
> I wonder if it depends on the font used. Is the selected font available on macOS?

I tried it with Dialog on macOS, it looked the same.

File: test/jdk/java/awt/font/TextLayout/TestJustification.java [hunk 1]
```diff
@@ -21,229 +21,241 @@
...
-    private static final String[] texts = {
-      "This is an english Highlighting demo.", "Highlighting",
-      "This is an arabic \u0627\u0628\u062a\u062c \u062e\u0644\u0627\u062e demo.", "arabic \u0627\u0628\u062a\u062c",
-      "This is a hebrew \u05d0\u05d1\u05d2 \u05d3\u05d4\u05d5 demo.", "hebrew \u05d0\u05d1\u05d2",
-      "This is a cjk \u4e00\u4e01\u4e02\uac00\uac01\uc4fa\uf900\uf901\uf902 demo.", "cjk",
-      "NoSpaceCJK:\u4e00\u4e01\u4e02and\uac00\uac01\uc4faand\uf900\uf901\uf902", "No",
-      "NoSpaceRoman", "Space"
-    };
+    private static JFrame createUI() {
+        JFrame frame= new JFrame("Test Text Justification");
```
[the remark this answers is not in the document]
Comment (honkar-jdk) [reply test/jdk/java/awt/font/TextLayout/TestJustification.java]: Updated.

**Review by aivanov-jdk — APPROVED** [review]

**Pull request closed without merging**